samples/04_gis_analysts_data_scientists/forecasting_monthly_rainfall_in_California_using_deeplearning_timeseries_model_from_arcgis_learn.ipynb

```diff
@@ -0,0 +1,2195 @@
+{
```

Comment:
<p>The most important step in this process is to determine if the timeseries sequence is autocorrelated. The ensure that our timeseries data can be modeled well, we must estimate the strength of correlation of the variable with its past data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>